stackit/internal/services/objectstorage/objectstorage_acc_test.go

```diff
@@ -0,0 +1,99 @@
+package objectstorage_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/stackitcloud/terraform-provider-stackit/stackit/internal/testutil"
+)
+
+// Bucket resource data
+var bucketResource = map[string]string{
+	"project_id":  testutil.ProjectId,
+	"bucket_name": fmt.Sprintf("acc-test-%s", acctest.RandStringFromCharSet(20, acctest.CharSetAlpha)),
+}
+
+func resourceConfig() string {
+	return fmt.Sprintf(`
+				%s
+
+				resource "stackit_objectstorage_bucket" "bucket" {
+					project_id = "%s"
+					bucket_name    = "%s"
+				}
+				`,
+		testutil.ObjectStorageProviderConfig(),
+		bucketResource["project_id"],
+		bucketResource["bucket_name"],
+	)
+}
+
+func TestAccObjectStorageResource(t *testing.T) {
```

Comment:
It is missing the TestCheckDestroy() function to confirm the resource has been deleted. Check the other acc tests